src/communication.h

```diff
@@ -74,10 +77,32 @@ std::pair<Property, SimpleVariant> processCanMessage(const std::vector<std::uint
         byte2 = msg[4U];
         property = static_cast<Property>(msg[2U]);
     }
-
+    
     const std::uint16_t value{static_cast<std::uint16_t>((byte1 << 8U) | byte2)};
+    
+    #if defined(PROPERTY_DEBUG_H)
+    //decode value with little endian
+    const std::uint16_t value_le{static_cast<std::uint16_t>((byte2 << 8U) | byte1)};
+    // decode receiver CAN ID 
+    std::uint16_t receiver_id = (((msg[0U] & 0xF0) *8)) + (msg[1U] & 0x0F);
+
+    //detect if it is a Read Write or broadcast message
+    if (msg[0] & 0x01) {
```

Comment:
with the code above
```
readWrite == 0x01
```

Reply:
At least that's how it was described by Juerg: [https://juerg5524.ch/data/Telegramm-Aufbau.txt]:
The lower byte of the fist 16bit message is either 1 for requests or 2 for a value change. Additionally the logs make /at least for me) sense like this.